python/tests/catalog/test_rest.py

```diff
@@ -524,9 +523,9 @@ def test_create_table_200(rest_mock: Mocker, table_schema_simple: Schema):
         schema=table_schema_simple,
         location=None,
         partition_spec=PartitionSpec(
-            spec_id=1, fields=(PartitionField(source_id=1, field_id=1000, transform=TruncateTransform(width=3), name="id"),)
+            PartitionField(source_id=1, field_id=1000, transform=TruncateTransform(width=3), name="id"), spec_id=1
```

Comment:
Looks like the mock causes the result to not match the request. We should start testing against the REST catalog servlet as soon as we can.